test/nodetool/utils.py

```diff
@@ -9,21 +9,41 @@
 import typing
 
 
-def check_nodetool_fails_with(
+def _do_check_nodetool_fails_with(
         nodetool,
         nodetool_args: tuple,
         nodetool_kwargs: dict,
-        expected_errors: typing.List[str]):
+        expected_errors: typing.List[str],
```

Comment:
it's not in the scope of this change, but since Python 3.9, the builtin types like `list` can also be used as generic types for type annotations. and f38 comes with Python 3.11, f39 Python 3.12. so we can just use `list[str]` here. see https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections

Reply:
I replaced it, I was changing this code anyway.